src/main/scala/is/hail/annotations/aggregators/RegionValueDownsampleAggregator.scala

```diff
@@ -23,6 +23,9 @@ class RegionValueDownsampleAggregator(nDivisions: Int) extends RegionValueAggreg
   def combOp(agg2: RegionValueAggregator) = combiner.merge(agg2.asInstanceOf[RegionValueDownsampleAggregator].combiner)
 
   def result(rvb: RegionValueBuilder) {
+    if (combiner.binToPoint == null) {
```

Comment:
Can you check if this also needs to be fixed in `DownsampleCombiner.toRes`? Add a test for this in `AggregatorsSuite`.